### DISCUSSION TITLE
文本检测微调后预测框比原始模型预测 的框要大

### DISCUSSION THREAD

**Discussion by huangguifeng** [PaddlePaddle/PaddleOCR#13207] [Show and tell]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/31758761/ad966d98-d78a-4ffb-b093-b5718c9e63db)  这是原始模型跑出来的结果 
![myocr](https://github.com/PaddlePaddle/PaddleOCR/assets/31758761/a49d09b8-dee3-48e7-b82f-ef8da3064e10)
后面这张图是微调z之后的结果，预测框明显大了很多。这是怎么回事呢

**Comment by huangguifeng** [comment]
求大佬指点

**Comment by huangguifeng** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/31758761/d970c92f-3075-4a6d-869b-0529233f229d)
推理时加了这个参数好像就正常了，，什么原因呢。为啥训练的模型预测框是 原来的1.5倍

**Reply by GreatV** [reply]
please refer to:  https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/FAQ.md#q%E8%AF%86%E5%88%AB%E6%A8%A1%E5%9E%8B%E6%A1%86%E5%87%BA%E6%9D%A5%E7%9A%84%E4%BD%8D%E7%BD%AE%E5%A4%AA%E7%B4%A7%E5%87%91%E4%BC%9A%E4%B8%A2%E5%A4%B1%E8%BE%B9%E7%BC%98%E7%9A%84%E6%96%87%E5%AD%97%E4%BF%A1%E6%81%AF%E5%AF%BC%E8%87%B4%E8%AF%86%E5%88%AB%E9%94%99%E8%AF%AF

**Reply by GreatV** [reply]
这个参数是检测后处理时控制文本框大小的，默认1.6，可以尝试改成2.5或者更大，反之，如果觉得文本框不够紧凑，也可以把该参数调小。

**Reply by huangguifeng** [reply]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/31758761/1fe5d913-d234-48be-aec3-6986635b8d13)
跟训练配置里这个参数有关系吗

**Reply by GreatV** [reply]
有关系
